### PR TITLE
Use CSS border for episode duration separator

### LINKED ISSUE #4034: a11y bug - episode list - the line before the timestamp should be a CSS border as this is presentation
**Describe the bug**
The line before the timestamp should be a CSS border as this is presentation. Using a character to achieve this is bad practice and also this looks odd/is not very clear/readable when there is no CSS 

e.g.
```html
<span aria-hidden=“true” class=“css-mro3c9 e1c0i89j0”>|</span>
```

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
CC @greenc05

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #4034

**Overall change:** 
Create a `VerticalSeparator` component that can be used when a vertical separator is needed for audio episodes

**Code changes:**
- Create `VerticalSeparator` component in index  file
- Use component in fixtures and README.ms

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-episode-list/src/fixtures.jsx [hunk 1]
```diff
@@ -206,7 +201,7 @@ export const renderEpisodes = ({
               <EpisodeList.Metadata>
                 <>
                   {' '}
-                  <StyledSpan aria-hidden>|</StyledSpan> {episode.date}
+                  <EpisodeList.VerticalSeparator /> {episode.date}
                 </>
               </EpisodeList.Metadata>
```
Comment (ryanmccombe) [inline packages/components/psammead-episode-list/src/fixtures.jsx]: I don't think we need the spaces here any more given the `VerticalSeperator` has its own spacing (from the padding and margin)

The spaces seem to be mis-aligning the divider: top is with spaces, bottom without:
Screen Shot 2020-12-11 at 10 07 42

File: packages/components/psammead-episode-list/src/index.jsx [hunk 2]
```diff
@@ -34,6 +34,16 @@ const StyledEpisodeListItem = styled.li`
   }
 `;
 
+const VerticalSeparator = styled.span`
+  padding-left: 8px;
+  margin-right: 8px;
+  border-right: 1px solid ${C_CLOUD_LIGHT};
```
Comment (ryanmccombe) [inline packages/components/psammead-episode-list/src/index.jsx]: This probably looks better if users resize text:
```suggestion
  padding-left: 0.5rem;
  margin-right: 0.5rem;
  border-right: 0.0625rem solid ${C_CLOUD_LIGHT};
```
(0.5rem could be the `GEL_SPACING` const instead)

**Review by ryanmccombe — APPROVED** [review]
👍

**Review by rebeccamcginn — APPROVED** [review]

**Comment by rebeccamcginn** [comment]
Ryan got in there first I'm afraid! You'll need to bump to 20

**Comment by greenc05** [comment]
Sorry I wasn't working at the end of last week, please could we reopen this, while it does the job it doesn't take into account good practice. 

It's not good practice to use an empty span just for presentation e.g. a border. It would be better to remove this span element and apply a CSS border on the container span, reducing the page weight and improving practice.

**Pull request merged**